### DISCUSSION TITLE
123网盘挂载出错

### DISCUSSION THREAD

**Discussion by liyangmoney** [AlistGo/alist#4991] [Q&A]
挂载后登录主页显示如下：
failed get objs: failed to list objs: json: cannot unmarshal string into Go struct field Files.data of type struct { InfoList []_123.File "json:\"InfoList\""; Next string "json:\"Next\"" }

**Comment by pongfcnkl** [comment]
别用123了，换一家网盘吧

**Reply by liyangmoney** [reply]
有好的推荐不

**Reply by pongfcnkl** [reply]
百度 115  阿里 夸克 三大运营商盘 等
